chrome/content/zotero/components/virtualized-table.jsx

```diff
@@ -290,11 +290,10 @@ class VirtualizedTable extends React.Component {
 		
 		this._columns = new Columns(this);
 		
-		this._rowHeight = props.rowHeight;
-		if (!this._rowHeight) {
-			this._rowHeight = props.defaultRowHeight || DEFAULT_ROW_HEIGHT;
+		this._rowHeight = props.rowHeight || DEFAULT_ROW_HEIGHT;
+		if (!props.disableFontSizeScaling) {
 			this._rowHeight *= Zotero.Prefs.get('fontSize');
-			if (Zotero.isMac && this._rowHeight > (props.defaultRowHeight || DEFAULT_ROW_HEIGHT)) {
+			if (Zotero.isMac && this._rowHeight > props.rowHeight || DEFAULT_ROW_HEIGHT) {
 				this._rowHeight -= 2;
 			}
```

Comment:
In relation to #2157 this conditional was supposed to get the 18px font size for macOS and 20px for Windows and Linux, but only when font scaling is 1 or higher. I think this is always true now for every OS, since `DEFAULT_ROW_HEIGHT` is always true and `&&` is evaluated before `||`.

Reply:
(Although my original condition may have been wrong too. It should have been `>=` there for it to work correctly)